src/main/webapp/i18n/de/programmingExercise.json

```
@@ -37,6 +37,9 @@
             "templateRepositoryUrl": "Vorlagen-Repository-Url",
             "testRepositoryUrl": "Test-Repository-Url",
             "solutionRepositoryUrl": "Lösungs-Repository-Url",
+            "auxiliaryRepositories": "Hilfs-Repositories",
+            "noCheckoutDirectorySet": "Es wurde kein Checkout-Verzeichnis gesetzt.",
+            "noCheckoutDirectorySetTooltip": "Da kein Checkout-Verzeichnis angegeben wurde, wurde das repository nicht automatisch zu einem Build-Plan hinzugefügt.",
```

```suggestion
            "noCheckoutDirectorySetTooltip": "Da kein Checkout-Verzeichnis angegeben wurde, wurde das Repository nicht automatisch zu einem Build-Plan hinzugefügt.",
```
But I am not sure here.